docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -98,25 +99,40 @@ into a file the kubeconfig that you will use as a user of kcp and
 KubeStellar.  Do not overwrite the kubeconfig file for your hosting
 cluster.  But _do_ update your `KUBECONFIG` envar setting or remember
 to pass the new file with `--kubeconfig` on the command lines when
-using kcp or KubeStellar.
+using kcp or KubeStellar. For example, you might use the following
+commands to fetch and start using that kubeconfig file; the first
+assumes that you deployed the core into a Kubernetes namespace named
+"kubestellar".
+
+```{.base}
+kubectl kubestellar get-external-kubeconfig -n kubestellar -o kcs.kubeconfig
+export KUBECONFIG=$(pwd)/kcs.kubeconfig
+```
+
+Note that you now care about two different kubeconfig files: the one
+that you were using earlier, which holds the contexts for your `kind`
+clusters, and the one that you just fetched and started using for
+working with the KubeStellar interface. The remainder of this document
+assumes that your `kind` cluster contexts are in `~/.kube/config`.
 
 ### Create SyncTarget and Location objects to represent the florin and guilder clusters
 
```

Comment:
I have chatted with some users that have already couple of clusters with different names and still want to follow along this example. It would be most helpful for them to use shell variables for the names of the clusters. For example, add:

```shell
export CLUSTER1=florin
export CLUSTER2=guilder
```

Then use $CLUSTER1 and $CLUSTER2 instead of florin and guilder in all the text and commands.

Reply:
Interesting idea. Remember the textual inclusions that we have among the doc source files. I would like to leave this to a separate PR.